app/models/application_form.rb

```diff
@@ -3,4 +3,8 @@
 class ApplicationForm < ApplicationRecord
   belongs_to :candidate
   has_many :application_choices
+
+  def personal_details_completed?
```

Comment:
Perhaps it's good to create a presenter in `ApplicationFormController#index` that can hold the logic about which form sections have been completed.

Reply:
Makes sense to me! We've added in an `ApplicationFormPresenter` now.